ting_material_details.field.inc

```diff
@@ -457,6 +463,36 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
             '#markup' => $entity->serieDescription,
           );
           break;
+
+        case 'ting_details_is_part_of_search':
+          $type = $uri = NULL;
+          foreach ($entity->getRelations() as $relation) {
+            $type = explode(':', $relation->getType());
+            if ( isset($type[1]) && $type[1] == 'isPartOfManifestation' ) {
+              $uri  = $relation->getURI();
+            }
+          }
+          $render['#prefix'] = '<span class="ting-material-detail ting-material-is-part-of">';
+          $render['#suffix'] = '</span>';
+          if (!is_array($val)) {
+            $val = array($val);
+          }
+          if ( $uri ) {
+            foreach ($val as $key => $detail) {
+              $render['part_of'][$key]['#theme'] = 'link';
+              $render['part_of'][$key]['#text'] = $detail;
+              $render['part_of'][$key]['#path'] = 'search/ting/' . $uri;
```

Comment:
the accept criteria for issue #28 is to have a link on the first isPartOfManifestation relation - so, we only need the one URI. The $val from ting_material_details_get_record_value() is an array with only one key/value pair (but that may change with issue#1109).
As for urlencoding: Drupal urlencodes the #path automatically as part of the rendering.